web/src/notifications.js

```diff
@@ -237,28 +253,20 @@ export function process_notification(notification) {
     let other_recipients;
     let title = message.sender_full_name;
     let msg_count = 1;
-    let notification_source;
 
     const topic = message.topic;
 
+    debug_notification_source_value();
```

Comment:
The `message` parameter is missing for a few commits.

Reply:
Thanks! Fixed!